src/Compilers/CSharp/Portable/Syntax/SyntaxNodeExtensions.cs

```diff
@@ -109,5 +109,29 @@ internal static SyntaxToken ExtractAnonymousTypeMemberName(this ExpressionSyntax
                 }
             }
         }
+
+        internal static TypeSyntax SkipRef(this TypeSyntax syntax, out RefKind refKind)
+        {
+            refKind = RefKind.None;
```

Comment:
Agreed. Will revise.

---
In reply to: [73057131](https://github.com/dotnet/roslyn/pull/12841#discussion_r73057131) [](ancestors = 73057131)